packages/slate/test/interfaces/Range/equals/equal.tsx

```diff
@@ -20,6 +21,7 @@ export const input = {
       path: [0, 1],
       offset: 0,
     },
+    placeholder: 1,
```

Comment:
Maybe call this `custom` or `foo` or something placeholder seems specific to your use case. So that it's obvious that it's custom data or better yet split it out into it's own `equals-user-marks.tsx` test file. That way both the just the anchor/focus comparison is tested and the one that has extra custom properties on it.